### PR TITLE
feat: add Community repo as content

### PR DESCRIPTION
**Pull request by aepfli**
We are adding the community repo as content.
This way we do have a better overview, and might structure our community guidelines more efficient.

### CODE REVIEW

**Comment by aepfli** [comment]
based on docusaurus our community repo has some issues - if this is something we want to continue with, i will fix them

```
- On source page path = /community/:
   -> linking to social-media (resolved as: /community/social-media)
   -> linking to ./LICENSE (resolved as: /community/LICENSE)

- On source page path = /community/branding-guidelines:
   -> linking to assets/logo/horizontal (resolved as: /community/assets/logo/horizontal)
   -> linking to assets/logo/horizontal/black (resolved as: /community/assets/logo/horizontal/black)
   -> linking to assets/logo/horizontal/white (resolved as: /community/assets/logo/horizontal/white)
   -> linking to assets/logo/stacked (resolved as: /community/assets/logo/stacked)
   -> linking to assets/logo/stacked/black (resolved as: /community/assets/logo/stacked/black)
   -> linking to assets/logo/stacked/white (resolved as: /community/assets/logo/stacked/white)
   -> linking to assets/logo/icon (resolved as: /community/assets/logo/icon)
   -> linking to assets/logo/icon/black (resolved as: /community/assets/logo/icon/black)
   -> linking to assets/logo/icon/white (resolved as: /community/assets/logo/icon/white)
```

**Comment by beeme1mr** [comment]
I think this is a great idea. Please feel free to adjust the Community markdown files to make the linter happy 😄

File: docusaurus.config.js [hunk 1]
```diff
@@ -139,6 +139,20 @@ const config = {
   ],
 
   plugins: [
+    [
+      '@docusaurus/plugin-content-docs',
```
Comment (beeme1mr) [inline docusaurus.config.js:143]: Nice, this looks like a better version of the custom one we wrote `externalContentPlugin`.

Comment (toddbaert) [reply docusaurus.config.js:143]: If it does the same job, I vote yes! If it's too time consuming though, you can open an issue or do it in a separate PR.

**Comment by toddbaert** [comment]
@aepfli I've approved https://github.com/open-feature/community/pull/159, hopefully that unblocks you here.

**Comment by aepfli** [comment]
another pr to improve the layouting - after this one, i think we can update the submodule and get this merged - https://github.com/open-feature/community/pull/160

**Comment by aepfli** [comment]
@beeme1mr @toddbaert this is now ready for review - be aware this was not about making the content beautiful, but making it visible.

image

another improvement i already see is removing the Readme.md from the asset folder :)

**Comment by beeme1mr** [comment]
Hey @aepfli, I've opened this PR to fix some typos and improve the overall structure of the community content in the docs.

https://github.com/open-feature/community/pull/162

Thanks for putting this together. I think this will be very helpful and end up being the best way to find community content.

**Pull request merged**